helix-term/src/ui/editor.rs

```diff
@@ -1109,6 +1109,7 @@ impl Component for EditorView {
         };
 
         match event {
+            Event::FocusGained | Event::FocusLost => todo!(),
```

Comment:
This is a new event on crossterm main that's handled in https://github.com/helix-editor/helix/pull/3178.